lib/services/cloud-build-service.ts

```diff
@@ -334,7 +335,8 @@ export class CloudBuildService extends CloudService implements ICloudBuildServic
 				flavorId: settings.projectSettings.flavorId,
 				additionalCliFlags: settings.additionalCliFlags,
 				useIncrementalBuild: !settings.projectSettings.clean,
-				userEmail: this.$nsCloudUserService.getUser().email
+				userEmail: this.$nsCloudUserService.getUser().email,
+				macOSImagePassword: await this.$nsCloudImageEncryptionService.getImagePassword(settings.projectSettings)
```

Comment:
I prefer having this calculated before constructing the object as it will be easier for debugging, but this is just a personal preference, not a merge stopper.